s/frontend/components/common/five-stars/namaka-review.ts

```diff
@@ -0,0 +1,26 @@
+import {renderFiveStarRating, stateBuddy} from './render-five-star-display.js';
+import fiveStarRatingCss from './render-five-star-display.css.js';
+import {mixinStyles} from '../../../framework/mixins/mixin-styles.js';
+import {html, LitElement} from 'lit';
+
+@mixinStyles(fiveStarRatingCss)
+export class NamakaReview extends LitElement {
+	render() {
+
+		customElements.whenDefined('namaka-review').then(() => {
+
+			let el: HTMLElement = document
```

Comment:
this line doesn't make sense, you can just do `this.shadowRoot.querySelector(".test")`